### ISSUE TITLE
NetAlarm

### ISSUE THREAD

**Issue by valepe** [psyciknz/CameraEvents#12]
Via NetAlarm it is possible to fire an alarm in port via network.

Do you think that is possible to add it?

I found that:
[https://gist.github.com/hdo/598f56a23d360164c3fbf60012902a88](url)

**Comment by psyciknz** [comment]
What is a net alarm?   Is that setting an alarm state in the nvr from http?  Rather than electrical?

**Comment by valepe** [comment]
Yes, with NetAlarm you can fire an alarm input port like an electrical switch.

In Dahua NVR, only an alarm in can fire an alarm out where you can connect a siren.
This is useful because your domotic hub can fire an alarm to the nvr if some events occur.

**Comment by psyciknz** [comment]
I’ll have a look in the http api which is what I do most if the communication over.

**Comment by psyciknz** [comment]
I’m not sure it’s possible without a completely new communication method.   Looks like it was removed when they removed their alarm server.

**Comment by valepe** [comment]
Bad news... Thanks anyway

**Comment by psyciknz** [comment]
I think look at one of those tools that some of the links post to to independently trigger it.   Or maybe look at an esp8266 hooked to one of your alarm in ports triggered by mqtt.

**Comment by valepe** [comment]
Yes, I'd look if there is a docker image of one of these solutions.
I prefer a wired solution to drive a siren :)
